src/interfaces/assistants_web/src/components/Agents/DiscoverAgents.tsx

```diff
@@ -1,28 +1,20 @@
 'use client';
 
-import { PropsWithChildren, useDeferredValue, useState } from 'react';
+import { useDeferredValue, useMemo, useState } from 'react';
 
 import { AgentPublic, ConversationWithoutMessages } from '@/cohere-client';
 import { DiscoverAgentCard } from '@/components/Agents/DiscoverAgentCard';
-import { Button, Icon, Input, Tabs, Text, Tooltip } from '@/components/Shared';
+import { Button, Input, Text } from '@/components/Shared';
 import { useListAgents } from '@/hooks/agents';
 import { useConversations } from '@/hooks/conversation';
+import { useSession } from '@/hooks/session';
 import { cn } from '@/utils';
 
-const tabs = [
-  <div className="flex items-center gap-2" key="company">
-    <Icon name="users-three" kind="outline" />
-    <Text>Company</Text>
-  </div>,
-  <div className="flex items-center gap-2" key="private">
-    <Icon name="profile" kind="outline" />
-    <Text>Private</Text>
-  </div>,
-];
+const GROUPED_ASSISTANTS_LIMIT = 15;
 
-const BASE_AGENTS: Array<AgentPublic & { isBaseAgent: boolean }> = [
+const BASE_AGENTS: Array<AgentPublic> = [
   {
-    id: '',
+    id: 'default',
```

Comment:
if you add the `default` as an id it will have another bg color than the agent list

Reply:
no it should not